### ISSUE TITLE
Plans for adding gen 9?

### ISSUE THREAD

**Issue by favna** [HybridShivam/Pokemon#11]
Hi, I was wondering if you were planning to add entries 906 (Sprigatito) to 1007 (Koraidon) for Generation 9.

And in the probably not all too distant future a dozen or so more for the 2 upcoming DLCs for Generation 9.

**Comment by favna** [comment]
Hi there @HybridShivam, first of all, best wishes for the new year!

The Indigo Disk DLC has been released for a couple of weeks now so I was wondering when you would be able to add the images?

**Comment by HybridShivam** [comment]
> Hi there @HybridShivam, first of all, best wishes for the new year!
> 
> The Indigo Disk DLC has been released for a couple of weeks now so I was wondering when you would be able to add the images?

Thank you!
I will try to add them as soon as I some free time: )

**Comment by favna** [comment]
@HybridShivam checking in here again, it's been a while

**Comment by HybridShivam** [comment]
> @HybridShivam checking in here again, it's been a while

This July for sure.

**Comment by EvolutionX-10** [comment]
@HybridShivam Eagerly waiting for the artworks! Thanks

**Comment by HybridShivam** [comment]
I have updated the web scrapper script.
As we are now over a 1000 mons, its better to shift to 4 digit IDs from 3 digit IDs, this might break some apps(including mine)  using this repo but this has to be done someday. Yes the change is breaking but can be fixed easily.
I don't want to break my app so it will be updated along with this repo causing some delay.

TLDR; Will update the repo with all kinds of images in a few days, have to compress/resize them and update my PokéDex app.

**Comment by HybridShivam** [comment]
This has been fixed with #14
All 1025 Species are available.
